back-end/models/Table.js

```diff
@@ -4,46 +4,87 @@ const Schema = mongoose.Schema;
 
 // Create Schema
 const TableSchema = new Schema({
-  location: {
+
+  ID:{
+    type: Number,
+    required:true
+
+  },
+  locationName: {
     type: PointSchema,
     required: true
   },
-  name: {
+  streetAddress: {
     type: String,
-    required: true
+    required: false
   },
-  description: {
+  city: {
     type: String,
-    required: true
+    required: false
   },
-  status: {
+
+  state: {
     type: String,
-    enum: ['submitted', 'approved', 'deleted', 'reported'],
-    default: 'submitted'
+    required: false
   },
-  tableType: {
+  zipcode: {
     type: String,
-    enum: ['fixed-height', 'adjustable', 'portable'],
-    required: true
+    required: false,
+    max:5
```

Comment:
Is there a way to do a minimum of 5 too? i think all zip codes should be exactly 5 digits

Reply:
I think it's fine without that